action_server/go-wrapper/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+//go:embed all:assets/*
+var content embed.FS
+
+func copyFiles(src, dest string) error {
+	files, err := content.ReadDir(src)
+	if err != nil {
+		return err
+	}
+
+	for _, file := range files {
+		// srcPath can't be constructed with filepath.Join because embed needs `/` as sep even on Windows
+		srcPath := fmt.Sprintf("%s/%s", src, file.Name())
+		destPath := filepath.Join(dest, file.Name())
+
+		if file.IsDir() {
+			err := copyFiles(srcPath, destPath)
+			if err != nil {
+				return err
+			}
+		} else {
+			fileContent, err := content.ReadFile(srcPath)
+			if err != nil {
+				return err
+			}
+
+			err = os.MkdirAll(filepath.Dir(destPath), 0755)
+			if err != nil {
+				return err
+			}
+
+			err = os.WriteFile(destPath, fileContent, 0755)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
+
+func main() {
+	var actionServerPath string
+	var executablePath string
+
+	// Read the version
+	versionData, err := content.ReadFile("assets/version.txt")
+	if err != nil {
+		fmt.Println("Error reading version.txt:", err)
+		os.Exit(1)
+	}
+	version := strings.TrimSpace(string(versionData))
+
+	// Determine the appropriate path based on the operating system
+	switch runtime.GOOS {
+	case "windows":
+		appDataDir := os.Getenv("APPDATA")
```

Comment:
Needs to be LOCALAPPDATA.
APPDATA points to `C:\Users\<user>\AppData\Roaming\robocorp` and we do not need this to be roaming.